examples/reagent/src/cljs/todomvc/helpers.cljs

```diff
@@ -0,0 +1,42 @@
+(ns todomvc.helpers
+  (:require [todomvc.session :as session]))
+
+(def enter-key 13)
+
+(def escape-key 27)
+
+(defn trim-title [title]
+  (clojure.string/trim title))
+
+(defn display-elem [bool] 
+  (if bool "inline" "none"))
+
+(defn todo-display-filter [completed display-type]
+  (condp = display-type
+    :completed completed
+    :active (not completed)
+    true))
+
+(defn create-todo-id []
+  (swap! session/todos-counter inc))
```

Comment:
Based on the spec(*), we use a UUID instead of an incrementing counter for the identifier.

(*) I actually have to check if we require it there, but it's the de facto way to implement it.